src/MessageResults/Dinosaur/Create.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\MessageResults\Dinosaur;
+
+class Create
+{
+    public function __construct(
+        public readonly string $id,
```

Comment:
Since we are using id instead of uuid in Module Messenger, we don't have yet the id of the species. For now, I'm using the species name.